README.md

```diff
@@ -331,55 +370,53 @@ In the server, `ctx` also exposes the same properties as a [Koa context](http://
-  * `acceptsEncoding: (...encodings: ...string) => boolean`
-  * `acceptsCharset: (...charsets: ...string) => boolean`
-  * `acceptsLanguage: (...languages: ...string) => boolean`
-  * `get: (name: String) => string` - returns a header
-  * `req: http.IncomingMessage` - [Node's `request` object](https://nodejs.org/api/http.html#http_class_http_incomingmessage)
-  * `res: Response` - [Node's `response` object](https://nodejs.org/api/http.html#http_class_http_serverresponse)
-  * `request: Request` - [Koa's `request` object](https://github.com/koajs/koa/blob/master/docs/api/request.md)
-  * `response: Response` - [Koa's `response` object](https://github.com/koajs/koa/blob/master/docs/api/response.md)
-  * `state: Object` - A state bag for Koa middlewares
-  * `app: Object` - a reference to the Koa instance
-  * `cookies: {get, set}`
-    * `get: (name: string, options: ?Object) => string` - get a cookie
-      * `name: string`
-      * `options: {signed: boolean}`
-    * `set: (name: string, value: string, options: ?Object)`
-      * `name: string`
-      * `value: string`
-      * `options: Object` - Optional
-        * `maxAge: number` - a number representing the milliseconds from Date.now() for expiry
-        * `signed: boolean` - sign the cookie value
-        * `expires: Date` - a Date for cookie expiration
-        * `path: string` - cookie path, /' by default
-        * `domain: string` - cookie domain
-        * `secure: boolean` - secure cookie
-        * `httpOnly: boolean` - server-accessible cookie, true by default
-        * `overwrite: boolean` - a boolean indicating whether to overwrite previously set cookies of the same name (false by default). If this is true, all cookies set during the same request with the same name (regardless of path or domain) are filtered out of the Set-Cookie header when setting this cookie.
-  * `throw: (status: number, message: ?string, properties: ?Object) => void` - throws an error
-    * `status: number` - HTTP status code
-    * `message: string` - error message
-    * `properties: Object` - is merged to the error object
-  * `assert: (value: any, status: ?number, message: ?string, properties)` - throws if value is falsy
-    * `value: any`
-    * `status: number` - HTTP status code
-    * `message: string` - error message
-    * `properties: Object` - is merged to the error object
-  * `respond: boolean` - set to true to bypass Koa's built-in response handling. You should not use this flag.
+  * `acceptsEncodings: (...encodings: ...string) => boolean` - check if encodings are acceptable
+  * `acceptsCharset: (...charsets: ...string) => boolean` - check if charsets are acceptable
+  * `acceptsLanguages: (...languages: ...string) => boolean` - check if langs are acceptable
+  * `get: (name: String) => string` - returns a header field
+  
+  </details>
+  
+  <details><summary>Properties below are aliases of Koa's response object</summary>
```

Comment:
It seems not very clear to me that these are properties under the `response` key. Can we add a bullet point here /w key name still? I'm not sure if this works, but something like:

```md
* `response: Object`: <details><summary>View koa response details</summary>...
```

Reply:
changed